From f7b2ec2aba334e37478c1a52b44bb2a67fc3017c Mon Sep 17 00:00:00 2001
From: Dirk Su <dirk.su@canonical.com>
Date: Thu, 5 Jan 2023 17:09:49 +0800
Subject: [PATCH] hwdb: Add Dell models that require ACCEL_LOCATION=base

(cherry picked from commit a554057f2b4bb6c2058ee9c97c407cc4baae9bae)

Related: RHEL-5950
---
 hwdb.d/60-sensor.hwdb | 2 ++
 1 file changed, 2 insertions(+)

diff --git a/hwdb.d/60-sensor.hwdb b/hwdb.d/60-sensor.hwdb
index aa827f97c5..3aa45b8289 100644
--- a/hwdb.d/60-sensor.hwdb
+++ b/hwdb.d/60-sensor.hwdb
@@ -348,6 +348,8 @@ sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0B0D:*
 sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0B11:*
 sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0C00:*
 sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0C02:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0C41:*
+sensor:modalias:platform:HID-SENSOR-200073:dmi:*svnDell*:sku0C42:*
  ACCEL_LOCATION=base
 
 # Dell Venue 8 Pro 3845
